app/domain/errors.rb

```diff
@@ -399,7 +399,14 @@ module AuthnJwt
               "1. Send the identity in the URL" \
               "2. Please add `token-app-property` variable and populate it" \
               "   with field in the decoded token that contains the identity",
-        code: "CONJ00075E"
+        code: "CONJ00175E"
+      )
+
+      JwtTokenClaimIsMissing = ::Util::TrackableErrorClass.new(
+        msg: "Claim '{0-attribute-name}' is missing from JWT token. " \
+             "Verify that you configured the host with permitted restrictions. " \
+             "In case of Compute Engine token, verify that you requested the token using 'format=full'",
+        code: "CONJ00176E"
```

Comment:
```suggestion
        code: "CONJ00076E"
```
?

Reply:
To avoid duplication

Reply:
https://github.com/cyberark/conjur/pull/2148#discussion_r627614858